recipes/ruptures/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ruptures" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1c4ae07622c26672ef6a113f6ac75b8c36f75bb9e649bf1bbc2c9afa23aee881
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python 
+requirements:
+  host:
+    - numpy
+    - pip
+    - python >=3.4
+    - scipy
+  run:
+    - numpy
+    - python >=3.4
+    - scipy
+
+test:
+  imports:
+    - ruptures
+    - ruptures.costs
+    - ruptures.datasets
+    - ruptures.detection
+    - ruptures.metrics
+    - ruptures.show
+    - ruptures.utils
+
+about:
+  home: https://github.com/deepcharles/ruptures
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: "Change point detection for signals, in Python"
+
```

Comment:
You can probably add `doc_url: http://ctruong.perso.math.cnrs.fr/ruptures-docs/build/html/index.html` here.